src/main/java/org/spongepowered/common/mixin/api/mcp/scoreboard/CriterionMixin_API.java

```diff
@@ -53,4 +74,8 @@
         }
         return this.spongeId;
     }
+
+    public Optional<TextColor> getTeamColor() {
```

Comment:
This implements the `@Interface`, so should use the prefix in that annotation - `criterion$`